packages/desktop-client/src/components/manager/WelcomeScreen.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import * as actions from 'loot-core/src/client/actions';
+import {
+  View,
+  Button,
+  Text,
+  P,
+  ExternalLink,
+} from 'loot-design/src/components/common';
+import { colors, styles } from 'loot-design/src/style';
+
+function WelcomeScreen({ createBudget, pushModal }) {
+  return (
+    <View
+      style={{
+        gap: 10,
+        maxWidth: 500,
+        fontSize: 15,
+        maxHeight: '100vh',
+        marginBlock: 20,
+      }}
+    >
+      <Text style={styles.veryLargeText}>Let’s get started!</Text>
+      <View
+        style={{
+          overflowY: 'auto',
+        }}
+      >
+        <P>
+          Actual is a personal finance tool that focuses on beautiful design and
+          a slick user experience.{' '}
+          <strong>Editing your data should be as fast as possible.</strong> On
+          top of that, we want to provide powerful tools to allow you to do
+          whatever you want with your data.
+        </P>
+        <P>
+          Currently Actual implements budgeting based on a{' '}
+          <ExternalLink
+            asAnchor
+            style={{ color: colors.p5 }}
+            href="https://actualbudget.github.io/docs/Budgeting/howitworks"
+          >
+            monthly envelope system
+          </ExternalLink>
+          . In the future, we’ll support multiple ways to do budgeting. We’re
+          also working hard on custom reports and a lot more things.
+        </P>
+        <P
+          style={{
+            fontWeight: 600,
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏IMO this should not be bolded. The new welcome page already has a lot of different text formatting options. For me as a user that feels overwhelming. I'm not sure what is more important in the text and what is less important.